flowzone.yml

```diff
@@ -1180,34 +1180,45 @@ jobs:
           echo "json=$(echo $response | jq -c .)" >> $GITHUB_OUTPUT
 
       # https://docs.github.com/en/rest/git/refs?apiVersion=2022-11-28#update-a-reference
-      - name: Update branch reference
+      - name: Update git references
         if: github.event.pull_request.merged == true && steps.create_commit.outputs.sha != ''
+        # some tags may not initially exist (e.g. production)
+        continue-on-error: true
```

Comment:
This should be a separate step so we don't ignore errors when updating the branch ref.

Reply:
Ignoring errors does not hide them from the job summary. This step will fail on 99% of repos and leave errors in the summary (even when the build continues).

I think this should be opt-in or we should make it possible to suppress the error only for the production tags.

Maybe we should add another env var to optionally ignore errors?

Something like `gh api ... ${{ env.APPEND_CMD }}` where the default is `` but for production we pass `|| exit 0`?